admob/src/ios/banner_view_internal_ios.mm

```diff
@@ -39,15 +40,24 @@
 
 Future<void> BannerViewInternalIOS::Initialize(AdParent parent, const char* ad_unit_id,
                                                AdSize size) {
-  const firebase::FutureHandle handle = CreateFuture(kBannerViewFnInitialize, &future_data_);
-  dispatch_async(dispatch_get_main_queue(), ^{
-    banner_view_ = [[FADBannerView alloc] initWithView:parent
+  FutureCallbackData* callback_data =
+      CreateFutureCallbackData(&future_data_, kBannerViewFnInitialize);
+  if(initialized_) {
+    future_data_.future_impl.Complete(callback_data->future_handle,
+                                      kAdMobErrorAlreadyInitialized,
+                                      "Ad is already initialized.");
```

Comment:
nit: AdMob is already initialized

Reply:
In this case it's the ad that's already initialized, not admob.